nilearn/input_data/tests/test_nifti_spheres_masker.py

```diff
@@ -178,3 +178,88 @@ def test_standardization():
     np.testing.assert_almost_equal(s.ravel(), data[1, 1, 1] /
                                    data[1, 1, 1].mean() * 100 - 100,
                                    )
+
+
+def test_nifti_spheres_masker_inverse_transform():
+    # Applying the sphere_extraction example from above backwards
+    data = np.random.random((3, 3, 3, 5))
+    img = nibabel.Nifti1Image(data, np.eye(4))
+    masker = NiftiSpheresMasker([(1, 1, 1)], radius=1)
+    # Test the fit
+    masker.fit()
+    # Transform data
+    s = masker.transform(img)
+    # Mask describes the extend of the masker's sphere
+    mask = np.zeros((3, 3, 3), dtype=np.bool)
+    mask[:, 1, 1] = True
+    mask[1, :, 1] = True
+    mask[1, 1, :] = True
+
+    inverse_map = masker.inverse_transform(s)
+    # Testing whether mask is applied to inverse transform, and has same length as 
+    # the masked data and data is preserved
+    assert_array_equal(get_data(inverse_map)[get_data(inverse_map) != 0], s[:, 0])
+    # No masking is applied - check dimensions with regard to mni template
+    assert_array_equal(inverse_map.shape[:3], [61, 73, 61])
```

Comment:
I'd rather not have the case without mask.

Reply:
I will implement the error (above) and test for it here then.